base/special/gamma.jl

```diff
@@ -134,347 +134,9 @@ lgamma{T<:Union{Float32,Float16}}(z::Complex{T}) = Complex{T}(lgamma(Complex{Flo
 
 gamma(z::Complex) = exp(lgamma(z))
 
-# Bernoulli numbers B_{2k}, using tabulated numerators and denominators from
-# the online encyclopedia of integer sequences.  (They actually have data
-# up to k=249, but we stop here at k=20.)  Used for generating the polygamma
-# (Stirling series) coefficients below.
-#   const A000367 = map(BigInt, split("1,1,-1,1,-1,5,-691,7,-3617,43867,-174611,854513,-236364091,8553103,-23749461029,8615841276005,-7709321041217,2577687858367,-26315271553053477373,2929993913841559,-261082718496449122051", ","))
-#   const A002445 = [1,6,30,42,30,66,2730,6,510,798,330,138,2730,6,870,14322,510,6,1919190,6,13530]
-#   const bernoulli = A000367 .// A002445 # even-index Bernoulli numbers
-
-"""
-    digamma(x)
-
-Compute the digamma function of `x` (the logarithmic derivative of [`gamma(x)`](@ref)).
-"""
-function digamma(z::Union{Float64,Complex{Float64}})
-    # Based on eq. (12), without looking at the accompanying source
-    # code, of: K. S. Kölbig, "Programs for computing the logarithm of
-    # the gamma function, and the digamma function, for complex
-    # argument," Computer Phys. Commun.  vol. 4, pp. 221–226 (1972).
-    x = real(z)
-    if x <= 0 # reflection formula
-        ψ = -π * cot(π*z)
-        z = 1 - z
-        x = real(z)
-    else
-        ψ = zero(z)
-    end
-    if x < 7
-        # shift using recurrence formula
-        n = 7 - floor(Int,x)
-        for ν = 1:n-1
-            ψ -= inv(z + ν)
-        end
-        ψ -= inv(z)
-        z += n
-    end
-    t = inv(z)
-    ψ += log(z) - 0.5*t
-    t *= t # 1/z^2
-    # the coefficients here are Float64(bernoulli[2:9] .// (2*(1:8)))
-    ψ -= t * @evalpoly(t,0.08333333333333333,-0.008333333333333333,0.003968253968253968,-0.004166666666666667,0.007575757575757576,-0.021092796092796094,0.08333333333333333,-0.4432598039215686)
-end
-
-"""
-    trigamma(x)
-
-Compute the trigamma function of `x` (the logarithmic second derivative of [`gamma(x)`](@ref)).
-"""
-function trigamma(z::Union{Float64,Complex{Float64}})
-    # via the derivative of the Kölbig digamma formulation
-    x = real(z)
-    if x <= 0 # reflection formula
-        return (π * csc(π*z))^2 - trigamma(1 - z)
-    end
-    ψ = zero(z)
-    if x < 8
-        # shift using recurrence formula
-        n = 8 - floor(Int,x)
-        ψ += inv(z)^2
-        for ν = 1:n-1
-            ψ += inv(z + ν)^2
-        end
-        z += n
-    end
-    t = inv(z)
-    w = t * t # 1/z^2
-    ψ += t + 0.5*w
-    # the coefficients here are Float64(bernoulli[2:9])
-    ψ += t*w * @evalpoly(w,0.16666666666666666,-0.03333333333333333,0.023809523809523808,-0.03333333333333333,0.07575757575757576,-0.2531135531135531,1.1666666666666667,-7.092156862745098)
-end
-
 signflip(m::Number, z) = (-1+0im)^m * z
 signflip(m::Integer, z) = iseven(m) ? z : -z
```

Comment:
oh but since you were importing the Base version in the package, you'll want to add these there instead